in_toto/runlib.py

```diff
@@ -570,6 +580,14 @@ def in_toto_run(name, material_list, product_list, link_cmd_args,
   if signature:
     signing_keyid = signature["keyid"]
     filename = FILENAME_FORMAT.format(step_name=name, keyid=signing_keyid)
+    
+    if metadata_directory is not None:
+      if not os.path.isdir(metadata_directory):
+        raise ValueError("--metadata-directory value is not a directory")
```

Comment:
Two general notes:
- a caller of this function might not know about the cli argument `--metadata-directory`. it's better to use the name used in this scope that is `metadata_directory`.
- it's more helpful user feedback if you include the value of the variable